chart/questions.yml

```diff
@@ -0,0 +1,44 @@
+categories:
+  - Knative
+  - faas
+  - serverless
+  - AWS
+questions:
+- variable: defaultImage
+  default: "true"
+  description: "Use default Docker images"
```

Comment:
nit: There are quite a few extraneous quotes in this file.

Reply:
The `questions.yaml` is not well documented so I used https://github.com/rancher/charts/tree/master/charts as a reference and across charts they are very consistent with the way quotes are used, i.e. the `label` tag does not use quotes whereas they are used everywhere else.

Reply:
I know some people do that because of coding habits, or because they use JSON a lot, but in reality YAML only requires quotes when the value is ambiguous (a number, a Boolean, a string starting with interpreted characters, ...).

It's fine as it is, I just wanted to point it out because we don't use quotes in other files across this repo.